tests/codeception/functional/Install/Content/FlexiblePageCest.php

```diff
@@ -151,4 +151,65 @@ public function testMobileMenu(FunctionalTester $I) {
+    $I->click('field_hs_page_hero_hs_sptlght_slder_add_more');
+    $I->waitForText('No media items are selected');
+    $I->canSee('Title');
+    $I->canSee('Height');
+    $I->canSee('Background Color');
+    $I->canSee('Image Alignment');
+    $I->canSee('Body');
+    // Populating spotlight #1.
+    $I->click('Add media', '.paragraph-type--hs-sptlght-slder');
+    $I->waitForText('Add or select media');
+    $I->dropFileInDropzone(dirname(__FILE__, 3) . '/logo.jpg');
+    $I->click('Upload and Continue');
+    $I->waitForText('Decorative Image');
+    $I->click('Save and insert', '.ui-dialog-buttonset');
+    $I->waitForText('The maximum number of media items have been selected');
+    $I->switchToIFrame('.cke_wysiwyg_frame');
+    $I->executeJS('document.getElementsByTagName("body")[0].innerHTML = "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>"');
+    $I->switchToIFrame();
+    $I->fillField(['name' => 'field_hs_page_hero[0][subform][field_hs_sptlght_sldes][0][subform][field_hs_spotlight_link][0][uri]'], 'http://google.com');
+    $I->fillField(['name' => 'field_hs_page_hero[0][subform][field_hs_sptlght_sldes][0][subform][field_hs_spotlight_link][0][title]'], 'Google Link');
+    $I->fillField(['name' => 'field_hs_page_hero[0][subform][field_hs_sptlght_sldes][0][subform][field_hs_spotlight_title][0][value]'], 'Spotlight #1 Title');
+    $I->scrollTo('#field-hs-page-components-hs-text-area-add-more');
+    $I->click('Add Spotlight');
+    $I->wait(1);
+    // Populating spotlight #2.
+    $I->click('//input[@class="js-media-library-open-button media-library-open-button button js-form-submit form-submit"][1]');
+    $I->waitForText('Add or select media');
+    $I->dropFileInDropzone(dirname(__FILE__, 3) . '/logo.jpg');
+    $I->click('Upload and Continue');
+    $I->waitForText('Decorative Image');
+    $I->click(['class' => "form-radio"]);
+    $I->click('Save and insert', '.ui-dialog-buttonset');
+    $I->waitForText('The maximum number of media items have been selected');
+    // Use javascript to select and populate correct iframe. 
+    $I->executeJS("var iframe = document.getElementsByClassName('cke_wysiwyg_frame')[1];
+      iframe.contentWindow.document.getElementsByTagName('body')[0].innerHTML = '<p>Aliquet porttitor lacus luctus accumsan tortor posuere ac.</p>';");
+    $I->fillField(['name' => 'field_hs_page_hero[0][subform][field_hs_sptlght_sldes][1][subform][field_hs_spotlight_link][0][uri]'], 'http://yahoo.com');
+    $I->fillField(['name' => 'field_hs_page_hero[0][subform][field_hs_sptlght_sldes][1][subform][field_hs_spotlight_link][0][title]'], 'Yahoo Link');
+    $I->fillField(['name' => 'field_hs_page_hero[0][subform][field_hs_sptlght_sldes][1][subform][field_hs_spotlight_title][0][value]'], 'Spotlight #2 Title');
+    $I->click('Save');
+    // Check spotlight 1.
+    $I->canSee('Spotlight #1 Title');
```

Comment:
@hectorlj Looks like the CI test is failing at this line and below. If I take a look at the artifact screenshot it appears the spotlight slider isn't appearing https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/472ddf5a-3a6a-4f76-ac26-eaa867026a6f/b2d54301-063a-4f05-b4bc-b0de6a53407a/0/var/www/html/artifacts/functional/FlexiblePageCest.testSpotlightSlider.fail.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOG5R5C2N5%2F20221020%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20221020T175942Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEIr%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIDRjGkIqpgcdDS5Mo7CYDqr6O%2F8eawDopKnWCA5UswA7AiEA%2BFj%2FT9c1erzROxhumX4eOmaI0%2F8gZLrTzHPHk8aibHkqqwIIYxADGgwwNDU0NjY4MDY1NTYiDHjizrw4bT1Okuo8uyqIAlXNl0%2Bg0UCUZ3PAh7rubnrzdBsU5PbHtjUm58tXRQ8G%2FOEx3ToGBgIYnMY22DgpXZ8Kq999a6kAbc9B4fe5nKYt8nMdEo1MltkNihQ8lfh6cS7m4NlxBNaPKerTvIZbKwiMB2ixypp4zF2safd1QC0s2xdAjNtfb8TruVEknSf0UJO7rC%2F%2B1ztLCYt7is2%2BaYXsOoveM4ybnd0fuw4Ng%2F3hdgBTyWtFTOa2ialqSvSjJNh1887B7%2B5CdcAlELmwHuu9PX2Pcq67Zi7N2QRKk2xazRsu9KvdaNsGdrFStz14IPy6cTnt%2BIhHcwSGaOD2bIbx7%2FpPxPSQ8BHojq6%2F1lq6rnXDQkE0hDD9lsaaBjqdAQ6VuTp6iA8fKQmtMwVR0aeUgIPlv9NaGCt78JJaeOAzgZMf%2BETmyKt5cMODbzsWJE01A6BpOfRLyA9akS5zYHtURF%2FyWvApN70sKeu8HR7hgDYUWPGQAxpbz1IZtDobuDeXVaOa7QOCQc%2FhM9JDWo1bHBKMhPdbJlrIRSva41WEhLLX9jHx%2BOZwT6d8UkohHNAw9qbePjt%2Fd2jwDs4%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=330b6fe8e692f42dcca2c5a33a18419b9c840997dd51c8cf181936cecae4f070

Reply:
Thanks @callinmullaney! Yeah I'm taking a look at this today

Reply:
Just fixed the issue, should be good to review now!